packages/EasyErrorHandler/src/Bridge/Laravel/Translator.php

```diff
@@ -14,19 +15,37 @@ final class Translator implements TranslatorInterface
      */
     private $decorated;
 
-    public function __construct(IlluminateTranslatorInterface $decorated)
+    /**
+     * @var string
+     */
+    private $translationNamespace;
+
+    public function __construct(IlluminateTranslatorInterface $decorated, string $translationNamespace)
     {
         $this->decorated = $decorated;
+        $this->translationNamespace = $translationNamespace;
     }
 
     /**
+     * @param string $message
```

Comment:
No need for these because we have type hint already. PHPDoc are required only for arguments/return that are not explicit about their content (e.g. array, iterable, ...)